Elizabeth_Theme/src/css/_custom.scss

```diff
@@ -0,0 +1,30 @@
+@import 'colors';
+@import 'components/buttons';
+
+@import 'layout/navigation';
+@import 'layout/welcome-page';
+@import 'layout/footer';
+
+@import 'components/map';
+
+body {
+  &.has-control-menu {
+
+    header {
+      margin-top: 0; 
+
+      @include media-breakpoint-down(sm) {
+        margin-top: 48px;
+      }  
+    }
+  }
+  
+  //RESPONSIVE BACKGROUND
```

Comment:
CSS comments need to be standardized like `/* ---------- Comment ---------- */`.

Reply:
CSS comments need to be in sentence case.